tauri/examples/communication/src-tauri/tauri.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable */
+
+/**
+ *  * THIS FILE IS GENERATED AUTOMATICALLY.
+ * DO NOT EDIT.
+ *
+ * Please whitelist these API functions in tauri.conf.json
+ *
+ **/
+
+/**
+ * @module tauri
+ * @description This API interface makes powerful interactions available
+ * to be run on client side applications. They are opt-in features, and
+ * must be enabled in tauri.conf.json
+ *
+ * Each binding MUST provide these interfaces in order to be compliant,
+ * and also whitelist them based upon the developer's settings.
+ */
+
+function s4() {
+  return Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1)
+}
+
+var uid = function () {
+  return s4() + s4() + '-' + s4() + '-' + s4() + '-' +
+    s4() + '-' + s4() + s4() + s4()
+}
+
+function ownKeys(object, enumerableOnly) { var keys = Object.keys(object); if (Object.getOwnPropertySymbols) { var symbols = Object.getOwnPropertySymbols(object); if (enumerableOnly) symbols = symbols.filter(function (sym) { return Object.getOwnPropertyDescriptor(object, sym).enumerable; }); keys.push.apply(keys, symbols); } return keys; }
+
+function _objectSpread(target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i] != null ? arguments[i] : {}; if (i % 2) { ownKeys(source, true).forEach(function (key) { _defineProperty(target, key, source[key]); }); } else if (Object.getOwnPropertyDescriptors) { Object.defineProperties(target, Object.getOwnPropertyDescriptors(source)); } else { ownKeys(source).forEach(function (key) { Object.defineProperty(target, key, Object.getOwnPropertyDescriptor(source, key)); }); } } return target; }
+
+function _defineProperty(obj, key, value) { if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }
+
+
+function _typeof(obj) { if (typeof Symbol === "function" && typeof Symbol.iterator === "symbol") { _typeof = function _typeof(obj) { return typeof obj; }; } else { _typeof = function _typeof(obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; }; } return _typeof(obj); }
+
+
+
+
+var __reject = function () {
+  return new Promise(function (_, reject) {
+    reject();
+  });
+}
+
+window.tauri = {
+  
+  invoke: function invoke(args) {
+    window.external.invoke(JSON.stringify(args));
+  },
+
+  
+  listen: function listen(event, handler) {
+    
+      
+        return __reject()
```

Comment:
seems like some features aren't available, can you build with `tauri build` to generate the file, and then remove it from the .gitignore?

Reply:
The rule in .gitignore is from `/src-tauri`. Do you mean removing this?

Reply:
The .gitignore inside tauri/examples/communication ignores `tauri.js`, I'm not sure how you commited it without removing that :D

Something is off because I don't have all these __reject on my generated tauri.js file.

About the index.tauri.html, you can ignore it.

Reply:
It's just `git add -f`. I have seen a few previous commit add to this directory, so I guess someone else was doing the same. In that case, it probably doesn't need to remove the rule.
Alright, I will just update the file `tauri.js`.